engine/source/elements/forint.F

```diff
@@ -188,11 +188,10 @@ SUBROUTINE FORINT(
       TYPE(ELBUF_STRUCT_), DIMENSION(NGROUP), TARGET :: ELBUF_TAB
       TYPE (NLOCAL_STR_) :: NLOC_DMG 
       TYPE (MATPARAM_STRUCT_), DIMENSION(NUMMAT) :: MATPARAM_TAB 
-      DOUBLE PRECISION,TARGET :: FBSAV6(12,6,DIMFB)
       TYPE (SUBSET_) , DIMENSION(NSUBS) :: SUBSET
       TYPE(H3D_DATABASE) :: H3D_DATA
       INTEGER, INTENT(IN) :: IFTHE, ICONDN
-      TYPE (SENSORS_) ,INTENT(IN) :: SENSORS
+      TYPE (SENSORS_) ,INTENT(IN) ,TARGET :: SENSORS
```

Comment:
Fortran standard does not allow this. Procedure that have a dummy argument with the TARGET keyword shall have an explicit interface.

Reply:
I have added an explicit interface for forint.F. By the way it helped to find a bug where variable FLG_KJ2 has changed declaration between INT and REAL between resol.F and forint.F